### ISSUE TITLE
Logs not appearing in Sentry

### ISSUE THREAD

**Issue by Jelle-elaut** [jovo-community/jovo-community-plugin-sentry#4]
Hi there! 
I tried installing the plugin using the given instructions, but no issues are coming through in Sentry.
I tried using the old `jovo-plugin-sentry` and with exactly the same code (except for the plugin config of course) the sentry connection is working correctly. 
I think it doesn't help to give you my code example, as it is exactly the same as in the readme.

When researching the issue, I found out that Jovo is always calling the On_Error() intent, but I don't have any error context in that place to manually log exceptions to Sentry.
Any help is appreciated! 
Thanks.

**Comment by rmtuckerphx** [comment]
Sorry, just saw this issue. Still not working?

**Comment by Jelle-elaut** [comment]
No, I'm using the old jovo plugin for sentry, which somehow works, but has almost the same implementation as this one
